.github/workflows/container_build_base.yml

```diff
@@ -3,8 +3,7 @@ name: Build base container
 
 "on":
   push:
-    branches:
-      - devel
+    branches: [ '**' ]
```

Comment:
I requested to remove this, but my comment disappeared. The stars are not needed. By default it will match all branches, so you can just remove the `branches` option everywhere.